### PR TITLE
Kubernetes: Generate cortx logs support bundle inside container.

### PR DESCRIPTION
**Pull request by gargpalak048**
# Problem Statement
-  Generate cortx logs support bundle inside container.

# Design
-  Created a sample pod which will bundle all the cortx logs under /var/log dir and generate a tarfile under /opt/ dir.
- sb_interface.py will provide an interface for user to request the support-bundle generation.
   example: - python3 sb_interface.py --generate

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: test/support_bundle/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM centos:7.9.2009
```
Comment (ujjwalpl) [inline test/support_bundle/Dockerfile:1]: Please add a README on how to use the tests.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: test/support_bundle/persistent_volumes/pv-claim.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```
Comment (ujjwalpl) [inline test/support_bundle/persistent_volumes/pv-claim.yaml:1]: Move these files into the main folder only to simplify. 
Also write a simple deploy.sh or such which calls these yaml files

Also use .yml as convention for consistency

File: test/support_bundle/sb_interface.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+import argparse
+import subprocess
+import shlex
+import tarfile
+import time
+
+from sb_config import PV_CLAIM_LIST, SB_FILE_PATH, SB_TAG
+
+
+class SSPLBundleError(Exception):
```
Comment (ujjwalpl) [inline test/support_bundle/sb_interface.py:12]: Remove SSPL. There is nothing called SSPL in CORTX for LC.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: test/support_bundle/persistent_volumes/pv-claim.yaml (hunk 2 above)
Comment (gargpalak048) [reply test/support_bundle/persistent_volumes/pv-claim.yaml]: Done.
Created a deploy_pvc.sh script will will create PV & PVCs
Renamed the deployment file to .yml

File: test/support_bundle/sb_interface.py (hunk 3 above)
Comment (gargpalak048) [reply test/support_bundle/sb_interface.py]: Ack.

**Comment by gargpalak048** [comment]
Support-bundle PR should be under cortx-utils repo,
PR against cortx-utils: https://github.com/Seagate/cortx-utils/pull/527
closing this PR.

**Pull request closed without merging**